docs/source/daml/intro/0_Intro.rst

```diff
@@ -9,6 +9,7 @@ DAML is a smart contract language designed to build composable applications on a
 In this introduction, you will learn about the structure of a DAML Ledger, and how to write DAML applications that run on any DAML Ledger implementation, by building an asset-holding and -trading application. You will gain an overview over most important language features, how they relate to the :ref:`da-ledgers` and how to use the DAML SDK Tools to write, test, compile, package and ship your application.
 
 This introduction is structured such that each section presents a new self-contained application with more functionality than that from the previous section. You can find the DAML code for each section `here <https://github.com/digital-asset/daml/docs/source/daml/intro/daml>`_ or download them using the DAML assistant. For example, to download the sources for section 1 into a folder called ``1_Token``, run ``daml new 1_Token daml-intro-1``.
+.. I either do not have access to git page or it doesn’t exist, so I had to guess what the files were called
```

Comment:
Are you referring to the fact that the link to `…/intro/daml` does not work? Good catch! That link is just broken. The correct link is https://github.com/digital-asset/daml/tree/master/docs/source/daml/intro. Do you want to update it?